src/store/reducers/activePage.js

```diff
@@ -0,0 +1,17 @@
+import * as actionTypes from '../actions/actionConstants'
```

Comment:
update page name to activePageReducer.js

Reply:
Updated.